docs/topics/rn-resolved-issues.adoc

```diff
@@ -16,15 +16,15 @@ For a complete list of all issues resolved in this release, see the list of link
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-3082[WINDUP-3082]
-|Eclipse plugin
-|A `NullPointerException` error occurs when browsing rules in the Rulesets view.
+|link:https://issues.redhat.com/browse/WINDUP-3125[WINDUP-3125]
+|ItelliJ IDEA plugin and Visual Studio Code extension
+|The *Apply QuickFix* menu option is still active after a QuickFix is applied.
 
-|link:https://issues.redhat.com/browse/WINDUP-3083[WINDUP-3083]
-|Eclipse plugin
-|When you click or double-click an issue in the Issue Explorer pane, the Issue Details pane is blank.
+|link:https://issues.redhat.com/browse/WINDUP-3134[WINDUP-3134]
+|ItelliJ IDEA plugin
+|After you run an analysis, the *Mark as Complete* menu option is not displayed.
```

Comment:
For release notes, it is best to start a sentence with the problem, not the circumstances.

However, since IntelliJ plugin is not documented, I question whether you should mention it in RN.